Documentation/gettingstarted/hubble-enable.rst

```diff
@@ -51,6 +51,22 @@ networking infrastructure in a completely transparent manner.
               --set global.hubble.relay.enabled=true \\
               --set global.hubble.ui.enabled=true
 
+        .. note::
+
+           For some environments, if you want to expose Hubble-UI to your protected network - 
+           we may recommend you to use reverse proxy with TLS and OAuth enabled. 
+           Ex: nginx-ingress-controller or aws-alb-ingress-controller
+           
+           For this purpose you may need to set Hubble-UI K8s service type to something other than default ClusterIP.
+           For this you can use param:
+
```

Comment:
To avoid duplicate `for this`, and in other places, we refer `helm option` instead of param.

```suggestion
           For this purpose you may need to set Hubble UI K8s service type to something other than default ClusterIP with ``helm`` option `hubble-ui.service.type`
```